### PR TITLE
Prepare for breaking change to UriResolver.

### PR DESCRIPTION
**Pull request by stereotype441**
Analyzer version 0.34.0 contains a breaking change to UriResolver: the
addition of a `clearCache` method - see
https://dart-review.googlesource.com/c/sdk/+/85726/1/pkg/analyzer/lib/src/generated/source.dart.

Even though we are not ready to upgrade to analyzer 0.34.0 yet, we can
prepare for the change by adding implementations of the method.  It's
not necessary for the implementations to do anything, since our
UriResolver implementations don't cache any URI resolution.

### CODE REVIEW

**Comment by stereotype441** [comment]
@scheglov FYI

File: bazel_codegen/lib/src/resolver/build_asset_uri_resolver.dart [hunk 1]
```diff
@@ -28,6 +28,9 @@ class BuildAssetUriResolver implements UriResolver {
     }
   }
 
+  @override
+  void clearCache() {}
```
Comment (kevmoo) [inline bazel_codegen/lib/src/resolver/build_asset_uri_resolver.dart]: Or extends? Then you can skip the blank override...

Comment (stereotype441) [reply bazel_codegen/lib/src/resolver/build_asset_uri_resolver.dart]: > Or extends? Then you can skip the blank override...

Done.  PTAL

**Comment by jakemac53** [comment]
Please also update changelogs/pubspecs

**Pull request merged**